app/views/invoices/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @invoices.uniq.each do |invoice| %>
```

Comment:
should there be a h1 tag with the merchant.name invoices?

Reply:
After reading the user story I was under the impression that only the one merchant who "owns" the invoices would be able to access this page so it would be redundant, but I could be wrong about that.

Reply:
I added the merchant name at the top and modified the tags around the invoices to h3 instead of h1.

Reply:
I think Kim already pointed this out, but we shouldn't be calling the whole Merchant class in a view and we should set an instance variable to carry it in